btree/btree.c

```diff
@@ -3703,6 +3710,21 @@ static int64_t btree_put_kv_tick(struct m0_sm_op *smop)
 		btree_callback_add(oi, lev->l_node, lev->l_idx);
 
 		node_unlock(lev->l_node);
+		return P_CAPTURE;
+	}
+	case P_CAPTURE: {
+		struct node_capture *arr = oi->i_capture;
+		struct slot          node_slot;
+		int                  i;
+
+		for (i = 0; i < BTREE_CALLBACK_CREDIT; i++) {
+			if (arr[i].nc_node == NULL)
+				break;
+			node_slot.s_node = arr[i].nc_node;
+			node_slot.s_idx  = arr[i].nc_idx;
+			node_capture(&node_slot, bop->bo_tx);
+		}
+
```

Comment:
It looks the change in `del_kv_tick` for P_CAPTURE are missing.

Reply:
It needs to get cover in different ticket EOS-22550, as some more changes are required for capturing transaction for Del operation.